<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Nagendra-Yoga" name="com.DollarDays.testCases.LoginPageTest" tests="5" failures="1" timestamp="13 Aug 2021 22:51:47 GMT" time="53.200" errors="0">
  <testcase name="TC_verifyLoginPageTitle" time="16.874" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyLoginPageMandatoryfields" time="0.248" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyForgotEmailPasswordLink" time="2.476" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyCreateAccountButton" time="11.732" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_logintoDD" time="21.870" classname="com.DollarDays.testCases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Could not Login to Dollardays with valid credentials.  Testcase failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Could not Login to Dollardays with valid credentials.  Testcase failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at com.DollarDays.testCases.LoginPageTest.TC_logintoDD(LoginPageTest.java:114)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- TC_logintoDD -->
</testsuite> <!-- com.DollarDays.testCases.LoginPageTest -->
